openstack/networking/v2/extensions/vpnaas/endpointgroups/doc.go

```diff
@@ -41,5 +41,16 @@ Example to List Endpoint groups
 	if err != nil {
 		panic(err)
 	}
+
+Example to Update an endpoint group
+
+	updateOpts := endpointgroups.UpdateOpts{
+		Name:        "updatedname",
+		Description: "updated policy",
```

Comment:
@simonre Looks like this needs updated to be a pointer-to-string. Since this is the last PR for Endpoints, do you mind fixing this before it's merged?